addons/project/views/project_views.xml

```diff
@@ -853,6 +882,15 @@
                                 <span class="o_stat_text">in Recurrence</span>
                             </div>
                         </button>
+                        <button name="action_subtask" type="object" class="oe_stat_button" icon="fa-tasks"
+                            attrs="{'invisible' : ['|', '|', ('allow_subtasks', '=', False), ('id', '=', False), ('subtask_count', '=', 0)]}" context="{'default_user_ids': user_ids, 'default_parent_id': id}">
+                            <div class="o_field_widget o_stat_info">
+                                <span class="o_stat_value">
+                                    <field name="subtask_count" widget="statinfo"/>
```

Comment:
We need to remove the label of this field since we have defined `o_stat_text`.
```suggestion
                                    <field name="subtask_count" widget="statinfo" nolabel="1"/>
```